quickstarts/deploy-vm/main.go

```diff
@@ -0,0 +1,159 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+
+	"github.com/Azure/azure-sdk-for-go/profiles/2017-03-09/network/mgmt/network"
+	"github.com/Azure/azure-sdk-for-go/profiles/2017-03-09/resources/mgmt/resources"
+
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/adal"
+	"github.com/Azure/go-autorest/autorest/azure"
+	"github.com/Azure/go-autorest/autorest/to"
+)
+
+type authInfo struct {
+	TenantID               string
+	SubscriptionID         string
+	ServicePrincipalID     string
+	ServicePrincipalSecret string
+}
+
+const (
+	resourceGroupName     = "GoVMQuickstart"
+	resourceGroupLocation = "eastus"
+
+	deploymentName = "VMDeployQuickstart"
+	templateFile   = "vm-quickstart-template.json"
+	parametersFile = "vm-quickstart-params.json"
+)
+
+var (
+	config = authInfo{
+		TenantID:               "",
+		SubscriptionID:         "",
+		ServicePrincipalID:     "",
+		ServicePrincipalSecret: "",
```

Comment:
YOu dont need to init the authinfo like this, with just `config = authInfo{}` is enough. Go sets the values to zero / empty

Reply:
Oh, I see what you are trying to do :) It might looks better with comments like `// add your creds here`.

---
In reply to: [167105445](https://github.com/Azure-Samples/azure-sdk-for-go-samples/pull/90#discussion_r167105445) [](ancestors = 167105445)

Reply:
✅